samples/grpc/local-drone-control-scala/src/main/scala/local/drones/DeliveryEvents.scala

```diff
@@ -0,0 +1,77 @@
+package local.drones
+
+import akka.actor.typed.{ ActorRef, ActorSystem, Behavior }
+import akka.persistence.Persistence
+import akka.persistence.query.typed.EventEnvelope
+import akka.projection.eventsourced.scaladsl.EventSourcedProvider
+import akka.projection.grpc.consumer.ConsumerFilter
+import akka.projection.grpc.consumer.scaladsl.GrpcReadJournal
+import akka.projection.r2dbc.scaladsl.R2dbcProjection
+import akka.projection.scaladsl.Handler
+import akka.projection.{ ProjectionBehavior, ProjectionId }
+import akka.util.Timeout
+
+/**
+ * Consume delivery events from the cloud and pass to the delivery queue actor
+ */
+object DeliveryEvents {
+
+  def projectionBehavior(
+      queueActor: ActorRef[DeliveriesQueue.Command],
+      settings: Settings)(
+      implicit system: ActorSystem[_]): Behavior[ProjectionBehavior.Command] = {
+    val projectionName: String = "delivery-events"
+
+    implicit val timeout: Timeout = settings.askTimeout
+
+    // initial consumer topic filter for location id
+    // FIXME no docs of setting up initial consumer filter, am I missing some API?
+    //       Async setup is a race condition but maybe ok? Does not seem to quite work, all or the wrong events are delivered
+    ConsumerFilter(system).ref ! ConsumerFilter.UpdateFilter(
+      // FIXME stream-id duplicated in config
+      "delivery-events",
+      // location id already is in the format of a topic filter expression
+      Vector(
+        ConsumerFilter.ExcludeRegexEntityIds(Set(".*")),
```

Comment:
this can use the new excludeAll, and include filter in GrpcReadJournal constructor

Reply:
Yep, but not until we made a milestone with those